### DISCUSSION TITLE
Bug Report: Crewman died from pressure with Slip Suit despite being ~400m before suit rated max. depth

### DISCUSSION THREAD

**Discussion by jonnymorrisuk** [FakeFishGames/Barotrauma#14028] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Security Officer crewman (player controlled, single player mode) put on a Slip Suit from a diving suit locker (marked with 'Ignore this', to prevent bots from using the Slip Suit), left via the airlock on my custom sub which I've used countless times before without problem.  Submarine depth was about 5,230m.  I immediately noticed the shimmering 'high pressure' visual effect, so I quickly turned around and tried to re-enter the submarine, I got inside the airlock and closed the outer door but my crewman still imploded.  

He was still wearing his Slip Suit so I know it didn't somehow accidentally come off.  I checked the depth rating of the Slip Suit - 5,700m.  Something didn't add up!

After having aborted the round and exited the game in annoyance, I later came back to do a few suit tests.  I constructed a basic diving suit (the green one) and left the sub wearing that, this time controlling my captain (the default character to start a round with).  To my susprise I didn't get the pressure effect, despite the rated depth for this suit being 4,000m and the sub was at 5,200 (had not moved since the start of the round).  I went back in and tested the other suits - again, I had to construct a Combat suit as I'd already ditched those in favour of the PUCS for my bots to wear all of the time (at least, when the put one on they just don't take them off again as they would with normal suits, and in dangerous waters it's prudent to always be wearing a suit anyway) - none suffered pressure problems at this depth of 5,230m.

I ended the round and went back to the main menu, then loaded the save game again and tested a basic diving suit again, this time I did get the pressure effect, and again my crewman imploded just as soon as I'd managed to get back inside the airlock and close the outer door.  Either it's just a coincidence or something odd is going on there too.

Later I have come back to the game and played the round as normal, with the usual things going on and completing missions that involve leaving the sub.  This time I only used PUCS or Abyss suits, no problems with pressure.  I have lost faith in the Slip Suits for now as it was all too quick and easy to lose a valued crewman to this weird pressure problem, and I have no way of knowing if or when it might happen again.

### Reproduction steps

1. Play a single player game to the Hydrothermal Wastes.
2. Leave the sub wearing a Slip Suit at a depth less than 5,700m (the maximum rated depth of the Slip Suit).
3. Wonder if the crewman will suffer from high pressure as if they were not wearing a suit at all.
4. Die from pressure anyway (if the problem occurs) despite making it back into the airlock and closing the outer door.

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by jonnymorrisuk** [comment]
My crewman didn't have any genetics or things like that, just standard equipment such as Rifle, Plasma Cutter, Harpoon Gun, Backpack, medical supplies, Security Officer's uniform, Riot Helmet, standard headset.  Probably had the usual minor ailments that don't heal normally, consistent with monster attacks and general fighting, though he would have been in very good general health before I took him on EVA.

**Comment by SomeRandomNoobKekeke** [comment]
I can confirm that in new 1.4.6.0 campaing slipsuits work fine at 5600m

No mods? Try to recraft your slipsuit, will it work?

**Reply by jonnymorrisuk** [reply]
I'm wondering if it's because this campaign is quite old now, probably started before the diving suit changes happened.  Will need to check something, would be useful to know the official depth of each suit for the current game version (my save might be a bit screwy in this regard).

**Reply by SomeRandomNoobKekeke** [reply]
That's exactly why i told you to recraft it

**Reply by SomeRandomNoobKekeke** [reply]
Regular Diving Suit - 4000m
Combat Diving Suit / Slipsuit - 5750m
PUCS - 6200m
Abyss Diving Suit / Respawn Diving Suit - 6650m
Exosuit / Funbringer - 7000m

**Reply by jonnymorrisuk** [reply]
> That's exactly why i told you to recraft it

I did test it again after reloading (no pressure problem), and I'd been using this exact same Slip Suit many times before this instance in recent days without problems.

**Reply by jonnymorrisuk** [reply]
I will recraft a Slip Suit as suggested and use it normally, see if the problem re-occurs.

**Comment by Regalis11** [comment]
We unfortunately have not been able to find a way to reproduce the issue: it probably requires some more specific circumstances to occur. Perhaps something to do with an old save from some specific earlier version of the game, or some combination of talents/afflictions/something that's interfering with the crush depths. We'll keep the ticket open though in case we or someone else manage to find some more clues for diagnosing the issue. Sorry for the inconvenience!
